scandium-core/src/main/java/org/eclipse/californium/scandium/config/DtlsConnectorConfig.java

```diff
@@ -117,9 +117,31 @@
 	 */
 	public static final long DEFAULT_STALE_CONNECTION_TRESHOLD = 30 * 60; // 30 minutes
 	/**
-	 * The default value for the {@link #retransmissionTimeout} property in milliseconds.
+	 * The default value for the {@link #retransmissionTimeout} property in
+	 * milliseconds.
+	 * 
+	 * @since 3.0 2s instead of 1s (following ACK timeout if RFC7252).
```

Comment:
(following ACK timeout ~if~ **in** RFC7252).   ?

Reply:
Yes.

Reply:
I add specific hrefs as well.

Reply:
:+1: